includes/modules/categories_tabs.php

```diff
@@ -2,37 +2,62 @@
 /**
  * categories_tabs.php module
  *
- * @copyright Copyright 2003-2022 Zen Cart Development Team
+ * @copyright Copyright 2003-2024 Zen Cart Development Team
  * @copyright Portions Copyright 2003 osCommerce
  * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
- * @version $Id: DrByte 2020 Dec 28 Modified in v1.5.8-alpha $
+ * @version $Id: v2.1.0-alpha3 $
  */
 if (!defined('IS_ADMIN_FLAG')) {
-  die('Illegal Access');
+    die('Illegal Access');
 }
-$order_by = " order by c.sort_order, cd.categories_name ";
+$link_class_active = 'category-top';
+$link_class_inactive = 'category-top';
+$span_wrap_active = 'category-subs-selected';
+$includeAllCategories = true;
```

Comment:
Use `??=`.  That'll enable BS4 to supply those values without modification to this core file.  Please keep the BS4 override suggestions; I'll get an issue opened for the template.